src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -308,6 +309,17 @@ void VfsCfApi::requestHydration(const QString &requestId, const QString &path)
     // of the placeholder with decrypted data
     if (record._isE2eEncrypted || !record._e2eMangledName.isEmpty()) {
         qCInfo(lcCfApi) << "Couldn't hydrate, the file is E2EE this is not supported";
+
+        QMessageBox e2eeFileDownloadRequestWarningMsgBox;
+        e2eeFileDownloadRequestWarningMsgBox.setText(tr("Download of End-to-End encrypted file failed"));
+        e2eeFileDownloadRequestWarningMsgBox.setInformativeText(tr("It seems that you are trying to download a virtual file that"
+                                                                   " is End-to-End encrypted. Implicitly downloading such files is not"
+                                                                   " supported at the moment. To workaround that issue, go to the"
```

Comment:
I think it is "to workaround _this_ issue" not "to workaround _that_ issue".

Reply:
Depends on the distance of the reader from the issue I guess :clown_face:

Sorry couldn't resist, the this/that distinction in English is so messy. :laughing:

Reply:
> Depends on the distance of the reader from the issue I guess 🤡
> 
> Sorry couldn't resist, the this/that distinction in English is so messy. 😆

"That's not an issue. THAT's an issue!" (c) Altered quote from some old movie 🤡